README.md

```diff
@@ -1,2 +1,55 @@
-# supervision-success-component
+# Supervision Success
+
 A React component powering a miniature web app for exploring the impact of success in supervision on the justice system at large
+
+## Contents
+
+1. [Input File Format](#input-file-format)
+
+## Input File Format
+
+Example of correct CSV could be found in [`params.csv`](/src/params.csv) file.
+
+First row of the table consists of next required params **in any order** (This params are used in application):
+
+- `state`
+- `newOffensePopulation`
+- `revocationA`
+- `revocationsTimescale`
+- `marginalCostPerInmate`
+- `totalCostPerInmate`
+- `checkpoint[n]`
+- `savings[n]`
+
+Checkpoint and according savings should be provided in such format:
```

Comment:
Let's reword this part to:


Checkpoint and savings columns model the changing amount of savings per person released from incarceration as the total number of people released passes certain checkpoints, i.e. as the number of people incarcerated decreases further, the amount of savings per person continues to change.

These columns should be provided in this format:
`checkpoint1`, `savings1` `checkpoint2`, `savings2` `...`, `checkpoint99`, `savings99`

You can provide as many checkpoint and savings columns as you'd like, but the number of each should be equal and there should be a 1:1 mapping from checkpoint _n_ to savings _n_.